.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -3,28 +3,84 @@ name: Bug report
 about: Report a bug for this project
 
 ---
+<!--
+  For community support and other discussions, you are welcome to visit us on our community channels listed at https://mailcow.github.io/mailcow-dockerized-docs/#community-support. For professional commercial support, please check out https://mailcow.github.io/mailcow-dockerized-docs/#commercial-support instead
+-->
 
-**README and remove me**
-For community support and other discussion, you are welcome to visit and stay with us @ Freenode, #mailcow
-Answering can take a few seconds up to many hours, please be patient.
-Commercial support, including a ticket system, can be found @ https://www.servercow.de/mailcow#support - we are also available via Telegram. \o/
+**Prior placing the issue, please check following:** *(fill out each checkbox with a `X` once done)*
+- [ ] I have understood that answers are voluntary and community-driven, and not kind of commercial support.
```

Comment:
I would instead say `and not commercial support`

Reply:
alternatively, you can use `and do not represent any kind of commercial support`